### PR TITLE
y NA issue, and report's functions

### PR DESCRIPTION
**Pull request by NedaJalali-codes**
1. good_indices in "aggregation.R file, censoring function" is removed from 

`overestimate[[paste0(colname, "_L")]] <- overestimate[[colname]][good_indices]`
to solve the y with NA values, related to issue #152  
I have tested two countries, SSD and BEN. Both ran successfully. 

2. Some functions in the report are commented out to solve the issues #160 and #161

### CODE REVIEW

File: Analysis/output/country_data_report.Rmd [hunk 1]
```diff
@@ -213,13 +213,13 @@ taxdat::plot_gam_fit_output_rates(config = params$config, cache = cache, cholera
 #### Ture underlying distribution of cases
 **True grid cases raster by time slices:**
 ```{r underlying distribution of cases,fig.cap=capFig("True underlying distribution of cases"), fig.height=5, fig.width=10}
-plot_true_grid_cases(config=params$config,cholera_directory=params$cholera_directory,cache=cache)
```
Comment (jkamins7) [inline Analysis/output/country_data_report.Rmd:216]: I think you maybe didn't mean to commit the changes to this file?

Comment (jkamins7) [reply Analysis/output/country_data_report.Rmd:216]: I think we should resolve the issue rather than commenting out the functions.

Comment (NedaJalali-codes) [reply Analysis/output/country_data_report.Rmd:216]: These are the functions that should only be in the testing report. For some reasons that I do not know, they are in the country_data_report, and you suggested to remove them.

Comment (jkamins7) [reply Analysis/output/country_data_report.Rmd:216]: I talked to Qulu and I think that the testing and non-testing reports got merged, and the testing plots should only happen if they are working. Can you try again (and let me and Qulu know if there are still problems with the reports)?

**Pull request merged**